server/data/nomisClientBuilder.js

```diff
@@ -141,9 +141,9 @@ module.exports = token => {
       return nomisPut({ path, body })
     },
 
-    async putChecksPassed(bookingId) {
+    async putChecksPassed({ bookingId, passed }) {
```

Comment:
would you want the `passed` to be defaulted?
i.e 
`{ passed = false }`
Or are you fine with `undefined` being a falsy?

Reply:
Good point. I'd rather a specific value was required because there's no obvious default behaviour. I'll change it so it's an error if no value supplied.